circle.yml

```diff
@@ -225,6 +225,38 @@ jobs:
            name: Test
            command: |
              test -f out/bin/ctags && ( file out/bin/ctags | grep -q 'ARM aarch64' )
+   centos6:
+     working_directory: ~/universal-ctags
+     docker:
+       - image: centos:6
+     steps:
+       - run:
+           name: Rearrange CentOS-Base.repo file
+           command: |
+             sed -i "s|#baseurl=|baseurl=|g" /etc/yum.repos.d/CentOS-Base.repo
+             sed -i "s|mirrorlist=|#mirrorlist=|g" /etc/yum.repos.d/CentOS-Base.repo
+             sed -i "s|http://mirror\.centos\.org/centos/\$releasever|https://vault\.centos\.org/6.10|g" /etc/yum.repos.d/CentOS-Base.repo
```

Comment:
`centos:6` and `centos:6.10` are actually the same image.

here we just change `mirror` to `vault` is ok:

```
sed -i 's|mirror|vault|g' /etc/yum.repos.d/CentOS-Base.repo
```

Reply:
Fixed. Thank you.